docs/get-started.md

```diff
@@ -4,14 +4,43 @@ title: Temporal documentation
 sidebar_label: Get started
 ---
 
+Welcome to Temporal's user documentation!
```

Comment:
I don't think we need this. I don't think people will think we're more genuine for it but I could be wrong. Also "user documentation" sounds demeaning or something.